app/src/components/fields/MultiAutocompleteField.tsx

```diff
@@ -93,45 +103,46 @@ const MultiAutocompleteField: React.FC<IMultiAutocompleteField> = (props) => {
     }
   };
 
-  const defaultChipDisplay = (option: any, renderProps: any, checkedStatus: any) => {
-    return (
-      <Box component="li" {...renderProps}>
-        <Checkbox
-          icon={<CheckBoxOutlineBlank fontSize="small" />}
-          checkedIcon={<CheckBox fontSize="small" color="primary" />}
-          style={{ marginRight: 8 }}
-          checked={checkedStatus}
-          disabled={(options && options?.indexOf(option) !== -1) || false}
-          value={option.value}
-          color="default"
-        />
-        {option.label}
-      </Box>
-    );
+  const getExistingValue = (existingValues: (number | string)[]): IMultiAutocompleteFieldOption[] => {
+    if (existingValues) {
```

Comment:
If `existingValues` is actually optional (hence this check), make the parameter optional:
```
const getExistingValue = (existingValues?: (number | string)[]): IMultiAutocompleteFieldOption[] => {
```
If `existingValues` isn't optional, remove this check (since it will always resolve `true`)

Reply:
Not sure the historical context for this check, but will add the ? for now, presuming the `if` was added for a good reason before.